beacon-chain/state/fieldtrie/field_trie_helpers.go

```diff
@@ -294,3 +328,32 @@ func handleBalanceSlice(val, indices []uint64, convertAll bool) ([][32]byte, err
 	}
 	return [][32]byte{}, nil
 }
+
+// emptyMVSlice specifies a type which allows a normal slice to conform
+// to the multivalue slice interface.
+type emptyMVSlice[V comparable] struct {
+	fullSlice []V
+}
+
+func (e emptyMVSlice[V]) Len(_ multi_value_slice.Identifiable) int {
+	return len(e.fullSlice)
+}
+
+func (e emptyMVSlice[V]) At(_ multi_value_slice.Identifiable, index uint64) (V, error) {
+	if index >= uint64(len(e.fullSlice)) {
+		var def V
+		return def, errors.Errorf("index %d out of bounds", index)
+	}
+	return e.fullSlice[index], nil
+}
+
+func (e emptyMVSlice[V]) Value(_ multi_value_slice.Identifiable) []V {
+	return e.fullSlice
+}
+
+func buildEmptyCompositeSlice[V comparable](values []V) multi_value_slice.MultiValueSliceComposite[V] {
+	return multi_value_slice.MultiValueSliceComposite[V]{
+		Identifiable:    nil,
+		MultiValueSlice: emptyMVSlice[V]{fullSlice: values},
+	}
+}
```

Comment:
It might make more sense to put this in the `container/multi-value-slice` package